### DISCUSSION TITLE
Vercel toolbar on custom domain

### DISCUSSION THREAD

**Discussion by Tom-Caz** [vercel/next.js#58551] [Help] [answered]
### Summary

Hello,

I was wondering if it is possible to get the Vercel toolbar to appear on a preview environment on a custom domain.

For example, my company has a `oursite.com` for production, and a `staging.oursite.com` preview for our staging branch (Through a third party registrar, not using Vercel domains). We really like the comment system the Vercel toolbar adds, but we'd like to be able to use it on our staging domain. Is this possible?

Thanks!

**Comment by nonameolsson** [comment]
We are facing the exact same situation. How come this one was closed without any comments?

**Reply by Tom-Caz** [reply, marked as answer]
Hey, sorry for not adding any comments. It ended up fixing itself after a few days, so I closed the discussion.

**Reply by nonameolsson** [reply]
Thanks!
Mind sharing how you solved it? 🙂

**Reply by Tom-Caz** [reply]
We didn't solve anything ourselves, after a few days the other domains just started working. Wish I could give advice, but we didn't have to do anything explicit on our end.
